iOS/Features/RewindJourney/Sources/RewindJourneyView/MSMusicView.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  MSMusicView.swift
+//  RewindJourney
+//
+//  Created by 전민건 on 11/22/23.
+//
+
+import UIKit
+
+import MSDesignSystem
+import MSLogger
+import MSUIKit
+
+final class MSMusicView: UIProgressView {
+    
+    // MARK: - Constants
+    
+    private enum Metrix {
+        
+        static var verticalInset: CGFloat = 8.0
+        static var horizonalInset: CGFloat = 12.0
+        static var cornerRadius: CGFloat = 8.0
+        
+        //albumart view
+        enum AlbumArtView {
+            static var height: CGFloat = 52.0
+            static var width: CGFloat = 52.0
+        }
+        //title view
+        enum TitleView {
+            static var height: CGFloat = 4.0
+            static var inset: CGFloat = 4.0
+            static var titleHight: CGFloat = 24.0
+            static var subTitleHight: CGFloat = 20.0
+        }
+        
+        //playtime view
+        enum PlayTimeView {
+            static var width: CGFloat = 67.0
+            static var verticalInset: CGFloat = 22.0
+            static var horizonalInset: CGFloat = 4.0
+            static var conponentsHeight: CGFloat = 24.0
+        }
+        
+    }
+    
+    private enum Default {
+        
+        //titleView
+        enum TitleView {
+            static var title: String = "Attention"
+            static var subTitle: String = "NewJeans"
+            static var defaultIndex: Int = 0
+        }
+        
+        //stackView
+        enum PlayTime {
+            static var time: String = "00 : 00"
+        }
+        
+    }
+    
+    // MARK: - Properties
+    
+    private var albumArtView = UIImageView()
+    private var titleView = UIStackView()
```

Comment:
```suggestion
    private let titleStack = UIStackView()
```

StackView는 프로퍼티 명에 Stack이 명시적으로 들어가면 좋을 것 같습니다.